### PR TITLE
Set up navigation & icons

### PR DESCRIPTION
**Pull request by ghost**
Adds the basic navigation and icons... still needs styling etc.

### CODE REVIEW

**Review by Kori-Kozicki — APPROVED** [review]
I think this UI is really good! It's good as a format but we should probably comment the classes, make tests eventually, and make more descriptive variable names. Also I had some issue with navigation not resolving on my end but that's probably android studio being dumb as usual. I think this would be a good base to merge into the main though, ie a good starting point

File: app/src/main/java/com/example/pygmyhippo/MainActivity.java [hunk 1]
```diff
@@ -2,23 +2,36 @@
...
 import androidx.appcompat.app.AppCompatActivity;
-import androidx.core.graphics.Insets;
-import androidx.core.view.ViewCompat;
-import androidx.core.view.WindowInsetsCompat;
+import androidx.navigation.NavController;
+import androidx.navigation.Navigation;
+import androidx.navigation.ui.AppBarConfiguration;
+import androidx.navigation.ui.NavigationUI;
+
+import com.example.pygmyhippo.databinding.ActivityMainBinding;
```
Comment (SuperCreativeUniqueUsername) [inline app/src/main/java/com/example/pygmyhippo/MainActivity.java:13]: databinding is not included in this pr

**Comment by SuperCreativeUniqueUsername** [comment]
looks awesome! i approve

**Review by SuperCreativeUniqueUsername — APPROVED** [review]
approved, this is an awesome base for all of us

**Pull request merged**